lib/account/models/profile.dart

```diff
@@ -51,6 +51,8 @@ class Profile extends Model {
 
   bool get isCurrentUser => id == SupabaseManager.getCurrentProfile()?.id;
 
+  static int get minAge => DateTime(1900).year;
```

Comment:
shouldn't that be maxAge?

Reply:
and maybe we should also add minAge, so that if we change that we don't have to change it in the factory

Reply:
yes that was actually what I meant? I think Cedric just misunderstood.